ffi/java-idsdk/src/main/java/com/iotics/sdk/identity/experimental/JWT.java

```diff
@@ -0,0 +1,53 @@
+package com.iotics.sdk.identity.experimental;
+
+import org.json.JSONObject;
+
+import java.text.SimpleDateFormat;
+import java.util.Base64;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class JWT {
+    private final String header;
+    private final String signature;
+    private final String payload;
+    private final String token;
+
+    public JWT(String token) {
+        this.token = token;
+        String[] chunks = token.split("\\.");
+        Base64.Decoder decoder = Base64.getDecoder();
+
+        this.header = new String(decoder.decode(chunks[0]));
+        this.payload = new String(decoder.decode(chunks[1]));
+        this.signature = chunks[2];
```

Comment:
Should this raise something other than `ArrayIndexOutOfBounds`?

Reply:
yeah - done... this - and the resolver client were built for utilitarian/debug purposes and not meant to be reused. 
the JWT class should really use a proper token parser